app/services/slack_notifier_service.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require "slack-notifier"
+
+class SlackNotifierService
+  include ServicePattern
+
+  def initialize(channel: "#twd_publish_register_tech", message:)
```

Comment:
Would it make sense to put this into `Settings`?

Reply:
Sure thing!